recipes/gunicorn/no_gaiohttp.patch

```diff
@@ -0,0 +1,174 @@
+diff --git a/gunicorn/workers/_gaiohttp.py b/gunicorn/workers/_gaiohttp.py
+deleted file mode 100644
+index cdce4be..0000000
+--- a/gunicorn/workers/_gaiohttp.py
```

Comment:
Remove the `a/` here to the patch applies cleanly everywhere. I know that conda-build changed this behavior and some versions don't need that, but we are pinning `conda-build` in some places.